library-and-framework-list-schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "https://raw.githubusercontent.com/oracle/graalvm-reachability-metadata/master/library-and-framework-support-schema.json",
+  "title": "Schema for the 'library-and-framework-list.json'",
+  "type": "array",
+  "default": [],
+  "items": {
+    "type": "object",
+    "required": [
+      "artifact",
+      "details"
+    ],
+    "properties": {
+      "artifact": {
+        "type": "string",
+        "pattern": "^[a-zA-Z0-9.-]+:[a-zA-Z0-9.-]+$",
+        "default": "",
+        "title": "The name of the artifact",
+        "description": "The artifact name in the groupId:artifactId format"
+      },
+      "description": {
+        "type": "string",
+        "default": "",
+        "title": "Short description of the library or framework"
+      },
+      "details": {
+        "type": "array",
+        "default": [],
+        "items": {
+          "type": "object",
+          "default": {},
+          "required": [
+            "minimal_version",
+            "metadata_locations",
+            "tests_locations",
+            "test_level"
+          ],
+          "properties": {
+            "minimal_version": {
+              "type": "string",
+              "default": "",
+              "title": "Minimal version for which this entry applies"
+            },
+            "maximal_version": {
+              "type": "string",
+              "default": "",
+              "title": "Maximal version for which this entry applies",
+              "description": "Maximum version for which this entry applies. If not defined, it is assumed to be supported for all versions starting from the minimum_version"
+            },
+            "metadata_locations": {
+              "type": "array",
+              "default": [],
+              "items": {
+                "type": "string",
+                "default": ""
+              },
+              "title": "Web URLs of provided metadata"
+            },
+            "tests_locations": {
+              "type": "array",
+              "default": [],
+              "items": {
+                "type": "string",
+                "default": ""
+              },
+              "title": "Web URLs to tests (sources, CI dashboards/configurations, ...)"
+            },
+            "test_level": {
+              "type": "string",
+              "enum": [
+                "Untested",
+                "Community tested",
+                "Fully tested"
+              ],
+              "default": "Untested",
```

Comment:
Considering the JSON file is meant for non-human consumption, maybe these should look more like symbols (`untested`, `community-tested`, `fully-tested`). At least, they should probably be lower-cased, and don't forget to update the example below.